yoyo.ini

```diff
@@ -1,6 +1,6 @@
 [DEFAULT]
 sources = migrations
-database = postgresql://postgres:postgres@localhost/postgres
+database = %(DATABASE_URL)s
```

Comment:
Happy to pull this one out into its own PR if you prefer. Was hitting an issue in my own setup because I was using a different postgres user